### PR TITLE
[bugfix] 테스트 데이터 정상출력되도록 수정

### PR DESCRIPTION
**Pull request by happy-dev-travel**
통합버전에서 테스트 데이터 출력 확인하였습니다

[input]
ADD,,,,99000101,A LEE,CL3,010-1234-5678,19920101,PRO
ADD,,,,99100201,B KIM,CL3,010-2341-5678,19910202,ADV
ADD,,,,00000301,C CHOI,CL2,010-3412-5678,19940403,EX
ADD,,,,18000301,D SEO,CL2,010-4123-5678,19991231,PRO
ADD,,,,21000301,E KANG,CL1,010-1234-6785,19990111,ADV
ADD,,,,99000102,AA LEE,CL4,010-1234-5679,19930101,EX
ADD,,,,99100202,BB KIM,CL4,010-2341-5679,19920202,PRO
ADD,,,,00000302,CC CHOI,CL3,010-3412-5679,19950403,ADV
ADD,,,,18000302,DD SEO,CL2,010-4123-5679,19991231,PRO
ADD,,,,21000302,EE KANG,CL1,010-1234-6789,20000111,ADV
ADD,,,,09000302,FF MOON,CL1,010-1234-6666,19901131,ADV
SCH,-p,,,employeeNum,00000301
DEL,-p,,,employeeNum,00000301
SCH,-p,,,employeeNum,00000301
MOD,-p,,,employeeNum,00000302,phoneNum,010-1111-2222
SCH,,,,birthday,19991231
DEL,,,,birthday,19991231
MOD,-p,,,cl,CL1,cl,CL2
DEL,-p,-f,,name,AA
MOD,-p,-l,,phoneNum,2222,certi,PRO
SCH,-p,-y,,birthday,1992

[console output]
SCH,00000301,C CHOI,CL2,010-3412-5678,19940403,EX
DEL,00000301,C CHOI,CL2,010-3412-5678,19940403,EX
SCH,NONE
MOD,00000302,CC CHOI,CL3,010-3412-5679,19950403,ADV
SCH,2
DEL,2
MOD,09000302,FF MOON,CL1,010-1234-6666,19901131,ADV
MOD,21000301,E KANG,CL1,010-1234-6785,19990111,ADV
MOD,21000302,EE KANG,CL1,010-1234-6789,20000111,ADV
DEL,99000102,AA LEE,CL4,010-1234-5679,19930101,EX
MOD,00000302,CC CHOI,CL3,010-1111-2222,19950403,ADV
SCH,99000101,A LEE,CL3,010-1234-5678,19920101,PRO
SCH,99100202,BB KIM,CL4,010-2341-5679,19920202,PRO

### CODE REVIEW

File: src/main/java/command/add/AddCommand.java [hunk 1]
```diff
@@ -54,12 +54,12 @@ else if(EmployeeInfo.get(2).equals("CL4"))
     }
 
 	private CERTI initialiseCerti(List<String> EmployeeInfo) {
-		CERTI certi = null;
-    	if(EmployeeInfo.get(5) == "ADV")
+		CERTI certi = CERTI.ADV;
```
Comment (smilesw6448) [inline src/main/java/command/add/AddCommand.java:57]: 여기서 바로 return CERTI.ADV 는 어떨까요?

**Review by smilesw6448 — APPROVED** [review]
변경사항점 확인했습니다.

**Review by bw2218 — APPROVED** [review]

**Review by jw451 — COMMENTED** [review]
확인했습니다.

File: src/main/java/command/add/AddCommand.java (hunk 1 above)
Comment (junheui27) [reply src/main/java/command/add/AddCommand.java:57]: 인라인 관점에서 바로 리턴해주는게 좋을것같네요.

**Pull request merged**